mirdata/datasets/good_sounds.py

```diff
@@ -0,0 +1,290 @@
+        destination_dir=".",
+    ),
+    "ratings": download_utils.RemoteFileMetadata(
+        filename="ratings.json",
+        url="https://zenodo.org/record/4588740/files/ratings.json?download=1",
+        checksum="b50b95fc7eb996b31a7cd290070f8059",
+        destination_dir=".",
+    ),
+    "sounds": download_utils.RemoteFileMetadata(
+        filename="sounds.json",
+        url="https://zenodo.org/record/4588740/files/sounds.json?download=1",
+        checksum="a60d90a964fd567ebc2e4b4e3f2990f2",
+        destination_dir=".",
+    ),
+    "takes": download_utils.RemoteFileMetadata(
+        filename="takes.json",
+        url="https://zenodo.org/record/4588740/files/takes.json?download=1",
+        checksum="318e840031397314907e7f9420f2abeb",
+        destination_dir=".",
+    ),
+    "all": download_utils.RemoteFileMetadata(
+        filename="good-sounds.zip",
+        url="https://zenodo.org/record/820937/files/good-sounds.zip?download=1",
+        checksum="2137bbb2d32c1d60aa51e1301225f541",
+        destination_dir=".",
+    ),
+}
+
+
+class Track(core.Track):
+    """GOOD-SOUNDS Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        track_id (str): track id of the track
+        audio_path (str): Path to the audio file
+
+    Cached Properties:
+        audio (tuple):
+        get_ratings_info (dict):  Some musicians rated some sounds in a 0-10 goodness scale for the user evaluation of the first
+            project prototype. Please read the paper for more detailed information.
+        get_pack_info (dict):
+        get_sound_info (dict): The entity containing the sounds annotations.
+        get_take_info (dict): A sound can have several takes as some of them were recorded using different microphones at the same
+            time. Each take has an associated audio file.
+    """
+
+    def __init__(
+        self,
+        track_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            track_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
+        self.track_id = track_id
+        self.audio_path = self.get_path("audio")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The track's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    @core.cached_property
+    def get_sound_info(self) -> dict:
+        return self._metadata()["sounds"][
+            str(self._metadata()["takes"][self.track_id]["sound_id"])
+        ]
+
+    @core.cached_property
+    def get_take_info(self) -> dict:
+        take_info = self._metadata()["takes"][self.track_id]
+        take_info["filename"] = self.audio_path
+        return take_info
+
+    @core.cached_property
+    def get_ratings_info(self) -> list:
```

Comment:
is it possible that a sound has empty ratings? 
it happened with track_id '2202': get_ratings_info returns an empty dict

Reply:
yes of course, only a little portion of sounds has ratings